src/index.css

```diff
@@ -1,13 +1,107 @@
-body {
+html, body {
   margin: 0;
+  height: 100%;
+}
+
+body {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+#root {
+  width: 100%;
+  height:100%;
+}
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.appContainer {
+  display: flex;
+  flex-direction: row;
+
+  height: 100%;
+}
+
+.towerList {
+  background-color: #FFD800;
+ 
+  padding: 16px;
+  max-width: 128px;
+
+  list-style: none;
+  margin: 0;
+}
+
+.towerIcon {
+  margin: 16px 0
+}
+.towerIcon:hover {
+  font-weight: bold;
+  cursor: pointer;
+}
+.towerIcon.selected {
+  background-color: #404040;
+  color:white;
+  padding: 8px;
+  padding-left: 16px;
+  width: 100%;
+
+  border-radius: 16px 0 0 16px;
+}
+
+.serverDetailsPanel {
+  color: white;
+  background-color: #404040;
+
+  padding: 16px;
+  margin: 0;
+}
+.channelList {
+  list-style: none;
+  padding: 0;
+}
+.channelIcon {
+  margin: 16px 0
+}
+.channelIcon:hover {
+  font-weight: bold;
+  cursor: pointer;
+}
+.channelIcon.selected {
+  background-color: #72716A;
+  font-weight: bold;
+  padding: 8px;
+  padding-left: 16px;
+  width: 100%;
+
+  border-radius: 16px 0 0 16px;
+}
+.channelIcon::before {
+  content: '#';
+}
+
+.messagePane {
+  background-color: #72716A;
+  margin: 0;
+  width: 100%;
+
+  padding: 16px 0 0 16px;
+  overflow: scroll;
```

Comment:
maybe instead of "scroll", make it "auto" so that it only allows scrolling if needed